source/agora/test/Base.d

```diff
@@ -1138,7 +1139,8 @@ public class TestAPIManager
 
         if (!no_txs)
         {
-            auto utxo_pairs = first_client.getSpendables(1.coins, OutputType.Payment);
+            auto utxo_pairs = first_client.getSpendables(1.coins, OutputType.Payment,
+                agora.script.Lock.LockType.Key);
```

Comment:
`agora.script.Lock.LockType.Key` is default to the `lock_type` parameter on the `getSpendables`, so can't we remove the last argument?

Reply:
Sadly no, `RemoteAPI` (LocalRest) generator does not work with default values.

Reply:
I see.